_layouts/prototype.html

```diff
@@ -3,39 +3,66 @@
 ---
 
 <div class="prototype">
-  <h2>{{ page.title }}</h2>
-  <p>{{ page.description }}</p>
-  
-  {% if page.prototype_url %}
-    <p><a href="{{ page.prototype_url }}">Try it now!</a></p>
-  {% endif %}
+  <div class="container">
+    <div class="col1">
+      <div class="row">
+        <p>
+          {{ page.category }} 
+          
+          {% if page.license == "MIT" or page.license == "mit" %}
 
-  {% if page.vimeo %}
-    <h3>Demo:</h3>
-    
-    <div class="resp-container">
-      <iframe class="resp-iframe" src="{{ page.vimeo }}" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>
+          <a href="{{ page.repo_url }}">
+            <img src="/images/mit-badge.svg" height="20" />
+          </a>
+          {% endif %}
+        </p>
+      </div>
+      
+      <h2>{{ page.title }}</h2>
+      <p>{{ page.description }}</p>
+      <div class="row">
+        {% if page.prototype_url %}
+        <a class="btn" href="{{ page.prototype_url }}">Try it!</a>
+        {% endif %} {% if page.repo_url %}
+        <a class="btn" href="{{ page.repo_url }}">Github Repo</a>
```

Comment:
Sometimes links like this flow into the footer somehow, like on this page: http://localhost:4000/prototypes/docker-nginx-webserver/